modules/opts.nix

```diff
@@ -67,12 +67,12 @@ in
             }:
             let
               varName = "nixvim_${luaVariableName}";
-              optionDefinitions = config.${optionName};
+              optionDefinitions = helpers.toLuaObject config.${optionName};
             in
-            lib.optionalString (optionDefinitions != { }) ''
+            lib.optionalString (optionDefinitions != "{ }") ''
```

Comment:
I see in matt's pr the string is `{}`, what do we generate with toLuaObject?

Reply:
We generate `"{ }"` which gets formatted by `luafmt` to `"{}"`.
Originally, I tried with `"{}"` and it was still showing.

Reply:
```
nix-repl> lib.x86_64-linux.helpers.toLuaObject {}
"{ }"
```